testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -98,10 +98,21 @@ protected void doTestHiveCoercion(HiveTableDefinition tableDefinition)
                 "tinyint_to_smallint",
                 "tinyint_to_int",
                 "tinyint_to_bigint",
+                "tinyint_to_string",
```

Comment:
Is it possible to split the commits into two where we add additional test cases to `BaseTestHiveCoercion` and ensure that they fail for unpartitioned tables and then apply the fix for orc type translator ?

Reply:
Done